tasks/section_01.yml

```diff
@@ -1,21 +1,163 @@
-#   See the License for the specific language governing permissions and
-#   limitations under the License.
-#
-
-  - include: section_01_level1.yml
-    tags:
-      - section1
-      - level1
+- name: 1.1 Ensure a separate partition for containers has been created (Scored)
+  debug: msg="*** This should be done by a separate playbook/process. Due to the ordering, this cannot be done during build-time"
+  tags:
+    - scored
+    - section1
+    - section1.1
+
+- name: 1.2 Ensure the container host has been Hardened (Not Scored)
+  debug: msg="*** This should be done by a separate playbook"
+  tags:
+   - notscored
+   - section1
+   - section1.2
+
+- name: 1.3 Ensure Docker is up to date (Not Scored)
+  package:
+    name: docker-ce
+    state: latest
+  tags:
+   - notscored
+   - section1
+   - section1.3
+
+- name: 1.4 Ensure only trusted users are allowed to control Docker daemon (Scored)
+  debug: msg="*** Not in a scope of this playbook"
+  tags:
+   - scored
+   - section1
+   - section1.4
+
+- name: Create auditd rules
+  lineinfile:
+    dest: /etc/audit/audit.rules
+    line: "-w {{ item }} -k docker"
+  notify: Reload auditd Rules
+  with_items:
+    - '/usr/bin/docker'
+    - '/var/lib/docker'
+    - '/etc/docker'
+    - '/lib/systemd/system/docker.service'
+    - '/lib/systemd/system/docker.socket'
+    - '/etc/default/docker'
+    - '/etc/docker/daemon.json'
+    - '/usr/bin/docker-containerd'
+    - '/usr/bin/docker-runc'
+  tags:
+   - scored
+   - section1
+   - section1.5
+   - section1.6
+   - section1.7
+   - section1.8
+   - section1.9
+   - section1.10
+   - section1.11
+   - section1.12
+   - section1.13
+
+# This forces handler to run
+- meta: flush_handlers
+  tags:
+    - scored
+    - section1
+    - section1.5
+    - section1.6
+    - section1.7
+    - section1.8
+    - section1.9
+    - section1.10
+    - section1.11
+    - section1.12
+    - section1.13
+
+- name: 1.5 Ensure auditing is configured for the Docker daemon (Scored)
+  command: bash -c "auditctl -l | grep /usr/bin/docker"
+  register: res
+  failed_when: res.rc == 1
+  changed_when: res.rc == 1
+  tags:
+   - scored
+   - section1
+   - section1.5
+
+- name: 1.6 Ensure auditing is configured for Docker files and directories - /var/lib/docker (Scored)
+  command: bash -c "auditctl -l | grep /var/lib/docker"
+  register: res
+  failed_when: res.rc == 1
+  changed_when: res.rc == 1
+  tags:
+   - scored
+   - section1
+   - section1.6
+
+- name: 1.7 Ensure auditing is configured for Docker files and directories - /etc/docker (Scored)
+  command: bash -c "auditctl -l | grep /etc/docker"
+  register: res
+  failed_when: res.rc == 1
+  changed_when: res.rc == 1
+  tags:
+   - scored
+   - section1
+   - section1.7
+
+- name: 1.8 Ensure auditing is configured for Docker files and directories - docker.service (Scored)
+  command: bash -c "auditctl -l | grep /lib/systemd/system/docker.service"
```

Comment:
Use shell module rather than command with bash -c?
Also couldn't you make these a with items.... rather than many separate rules?

Reply:
The separation is so someone who isn't me can work down the list and tick things off